src/Components/login/LoginPage.tsx

```diff
@@ -1,10 +1,12 @@
 import React, { ReactElement } from "react";
 import { Grid, makeStyles, Paper, CssBaseline, StyleRules } from "@material-ui/core";
+import { useSelector } from "react-redux";
 import Login from "./Login";
 import pic from "../../assets/GmapsRevature.png";
+import Register from "./Register";
```

Comment:
Rename file to LandingPage to clarify that this container component is more than just the login page. It also holds registration. We need to maintain self-documenting clear names, especially as the project scales.